saugardev/react-ui-components
# Build/2/develop_landing_page_header

# Overview
This PR is attributed to ticket #[2](https://github.com/Raisin-DAO/raisin-app/issues/2) on the Raisin App project. This PR adds a landing page header as wireframed in the ticket.

Part of this ticket includes Tailwind config setup. It can be used to live update Storybook with Tailwind classes in dev. Packages of postcss and autoprefixer were mandatory additions for Tailwind. Explanation of how to use this to live update Storybook in dev was added to the ReadME. 

The PageHeader component was added, which includes Navigation item sub components. General base styles for active, and hover pseudo classes were added to show a baseline functionality. Style + color are expected to be changed and refined. 

Storybook stories were changed to CSF3, the recommended format for stories - [docs](https://storybook.js.org/blog/storybook-csf3-is-here/).

# How to test
1. Pull down this branch
2. Run storybook locally
3. Open the PageHeader component and confirm it meets criteria shown in ticket 

# Notes:
- The header is not responsive for all screen sizes
- Font style should be updated
- LOGO text and square icon are being used as placeholders
